docs/pages/eas/workflows/syntax.mdx

```diff
@@ -384,6 +384,33 @@ jobs:
       prod: boolean # optional
 ```
 
+#### `fingerprint`
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    type: fingerprint
+    # @end #
+```
+
+Calculates fingerprint of iOS and Android builds.
+
+> **warning** This job type only supports CNG (managed) workflows. If you commit your `/android` or `/ios` directories the fingerprint job won't work.
```

Comment:
```suggestion
> **warning** This job type only supports CNG (managed) workflows. If you commit your **android** or **ios** directories, the fingerprint job won't work.
```

Let's make the [directory name bold as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text). Also, no need to prefix directory names with `/`. Without it, it's a common pattern we everywhere in docs